deploy/helms/event-collector/Chart.yaml

```diff
@@ -0,0 +1,9 @@
+name: event-collector
+version: 0.0.1
+appVersion: 1.7.1
+description: Legion event collector
+maintainers:
```

Comment:
Should *Legion team*  with *legion-dev@googlegroups.com* be mentioned here?

Reply:
I'm not sure about team email, @akharlamov which one I should provide?

Reply:
Yes, looks right to me.

Reply:
OK, I'll update it to `Legion team` with `legion-dev@googlegroups.com`